addons/hr_expense/tests/test_expenses.py

```diff
@@ -975,3 +975,33 @@ def test_create_report_name(self):
             sheet_names,
             "The report name should be 'New Expense Report, paid by (employee|company)' as a fallback",
         )
+
+    def test_update_expense_price_on_product_standard_price(self):
+        """
+        Tests that updating the standard price of a product will update all the un-submitted
+        expenses using that product as a category.
+        """
+        expense_category = self.env['product.product'].create({
+            'name': 'Expense Category',
+            'default_code': 'EC',
+            'standard_price': 100.0,
+        })
+        expense = self.env['hr.expense'].create({
+            'employee_id': self.expense_employee.id,
+            'name': 'Expense 1',
+            'product_id': expense_category.id,
+            'total_amount': 1,
+        })
+        expense_sheet = self.env['hr.expense.sheet'].create({
+            'name': 'Expenses paid by employee',
+            'employee_id': self.expense_employee.id,
+            'expense_line_ids': expense,
+        })
+        expense_category.write({'standard_price': 120.0})
+        self.assertEqual(expense.total_amount, 120.0,
```

Comment:
```suggestion
        })
        expense_category.standard_price = 120.0
        self.assertEqual(expense.total_amount, 120.0,
```
Can't blame you, I do that a lot